.gitignore

```diff
@@ -43,3 +43,4 @@ package/autoyast2-*.bz2
 /.yardoc
 /coverage
 /test-driver
+ 
```

Comment:
why?

Reply:
No idea :) This comes from https://github.com/yast/yast-autoinstallation/commit/17450d641cb30959ce331cbf6446b95d5952350b and I tried to _modify changes_ as less as possible.

That said, we can get rid of this line if you want.